xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEFilter.cpp

```diff
@@ -192,7 +192,7 @@ int CActiveAEFilter::ProcessFilter(uint8_t **dst_buffer, int dst_samples, uint8_
     frame->format = m_sampleFormat;
 
     result = avcodec_fill_audio_frame(frame, channels, m_sampleFormat,
-                             src_buffer[0], channels * src_samples * av_get_bytes_per_sample(m_sampleFormat), 0);
+                             src_buffer[0], channels * src_samples * av_get_bytes_per_sample(m_sampleFormat), 1);
```

Comment:
In short: replacing 1 with 16 will work, too, but only by chance for 4 byte format... If Format is 3 Byte and two channel this method will fail, too.